pkg/audit/cursor.go

```diff
@@ -49,64 +54,79 @@ func (cursor *Cursor) NextStripe(ctx context.Context) (stripe *Stripe, err error
 	var path storj.Path
 	var more bool
 
-	pointerItems, more, err = cursor.pointerdb.List("", cursor.lastPath, "", true, 0, meta.None)
-	if err != nil {
-		return nil, err
+	var attempts = 0
+	for attempts < maxListAttempts {
+		pointerItems, more, err = cursor.pointerdb.List("", cursor.lastPath, "", true, 0, meta.None)
+		if err != nil {
+			return nil, err
+		}
+
+		// keep track of last path listed
+		if !more {
+			cursor.lastPath = ""
```

Comment:
does it make sense to make another attempt if there is nothing else to list?

Reply:
setting cursor.lastPath to `""` means that pointerdb will start listing from the beginning. are you saying we shouldn't start listing from the beginning? or that I need another check?

Reply:
hmm my understanding was wrong. Code is ok and we should start listing from the beginning.